README.md

```diff
@@ -60,3 +60,22 @@ dnf update -y && dnf install -y sudo git python3 python3-distro gcc-toolset-11
 scl enable gcc-toolset-11 bash
 make # or "make server" for packages specific to building the iRODS server.
 ```
+
+# FAQ
+
+### Q. Can I build an externals package using a different repository?
+Yes. For the externals package of interest, simply set the `"git_repository"` property to the URL of the repository to clone. If this property is not present, it means the build will clone the forked repository under https://github.com/irods. The build expects that the clone will have a directory name matching the name of the forked project under https://github.com/irods.
```

Comment:
Should we mention `versions.json` to make it clear where the property being set is located? I didn't pick up on that the first time reading this

Reply:
good point. will add that in.

Reply:
updated.

thoughts?

Reply:
Looks good to me!